pypredict/trades.py

```diff
@@ -55,11 +58,10 @@ async def recv_and_publish(self, uri):
                     while True:
                         message = await websocket.recv()
                         for event in self.message_to_events(json.loads(message)):
-                            await self.ensign.publish(topic_id, event, on_ack=handle_ack, on_nack=handle_nack)
+                            await self.ensign.publish(self.topic, event, on_ack=handle_ack, on_nack=handle_nack)
             except websockets.exceptions.ConnectionClosedError as e:
-                # TODO: Make sure reconnect is happening for dropped connections.
-                print(f"Websocket connection closed: {e}")
-                await asyncio.sleep(1)
+                logging.error(f"Websocket connection closed: {e}")
+                continue
```

Comment:
Ok if I'm reading this correctly `continue` in this instance is the same as falling through to the next iteration right? So maybe it's not the cause of the app not publishing, but I think this is more clear anyway. Maybe the logging will help us figure out what's going on.